src/features/make_features.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+
+def make_predictors(df, date, hood):
+    """ Create predictor variables.
+    TODO: this function should resolve issue #27
+    https://github.com/the4thparadigm/denvercrime/issues/27
+    TODO: write a docstring explaining your approach
+    TODO: submit a PR with your changes to this file
+
+    Parameters:
+        df (DataFrame): crime data
+        date (datetime.date): value for df['REPORTED_DATE']
+        hood (str): value for df['NEIGHBORHOOD_ID']
+
+    Returns:
+        (Series): Counts of different combinations of date, offense type, and location
```

Comment:
we could consider also returning a pandas series of strings which give human readable names for each feature. Okay to leave out for now though.